app/poros/trivia_question.rb

```diff
@@ -0,0 +1,25 @@
+class TriviaQuestion
+  attr_reader :category,
+              :type,
+              # :difficulty,
+              :question,
+              :correct_answer,
+              :incorrect_answers,
+              :answers
+
+  def initialize(info)
+    @category           = info[:category]
+    @type               = info[:type]
+    # @difficulty         = info[:difficulty]
```

Comment:
might be worth adding a top level comment with a brief explanation about why difficulty is commented out for now